### ISSUE TITLE
Add command not working?

### ISSUE THREAD

**Issue by tbdabbholm** [hallidev/delta-bot-four#8]
I've been trying to use the add command but it doesn't seem to be working. I don't get an error message or a confirmation message.

**Comment by hallidev** [comment]
I'm continuing with reliability improvements so that comments and private messages won't be missed.

I'll be deploying a change to private messages in a bit, and it should pick up any unread private messages when it starts.

Improvements to comments and edits will be coming in the next day or two.

**Comment by hallidev** [comment]
An update to private messages has been deployed. The hope is that PM reliability will be great improved. I'll keep this open for a few days. Please let me know if there are issues with DeltaBot not processing / replying to PMs

**Comment by ansuz07** [comment]
It doesn't seem to be processing add requests today.

**Comment by hallidev** [comment]
Very surprising. I'll take a look and post back here - perhaps 2 or so hours.

**Comment by hallidev** [comment]
I restarted the bot, but there' s a problem when the backlog gets too big. There were so many commands backed up that we blew way past our rate limit for the bot and I had to restart it a second time. There will be a number of commands that were missed unfortunately.

This will become less of an issue once it stops missing comments and PMs, and I'm working on another update for comment reliability.

I have to admit though - I thought I had the PM issue licked, and this proves that there's still work to be done there.

Thanks for hanging in there with this. I will get this thing stable one way or another.

On the plus side, everything outside of the monitoring has proven stable at this point, so once monitoring incoming comments / edits / pms is all the way there, we should be good.
